site/content/docs/Repositories/_index.md

```diff
@@ -24,7 +23,7 @@ Kopia allows you to save your [encrypted](../features/#end-to-end-zero-knowledge
 * Dropbox, OneDrive, Google Drive, and all cloud storage supported by [Rclone](#rclone) 
   * Rclone is a (free and open-source) third-party program that you must download and setup seperately before you can use it with Kopia
   * Once you setup Rclone, Kopia automatically manages and runs Rclone for you, so you do not need to do much beyond the initial setup, aside from enabling Rclone's self-update feature so that it stays up-to-date
-  * Kopia's Rclone support is experimental and all the cloud storage supported by Rclone has not been tested to work with Kopia; Kopia has been tested to work with Dropbox, OneDrive, and Google Drive through Rclone
+  * Kopia's Rclone support is experimental: all the cloud storage supported by Rclone has not been tested to work with Kopia, and some may not work with Kopia; Kopia has been tested to work with Dropbox, OneDrive, and Google Drive through Rclone
```

Comment:
nit: "not all cloud storage ... has been tested" instead of "all the cloud storage ... has not been tested"